exporters/elasticsearch-exporter/README.md

```diff
@@ -155,18 +155,21 @@ exporters:
         event: true
         rejection: false
 
-        deployment: false
+        deployment: true
         process: true
         error: true
         incident: true
         job: true
         jobBatch: false
-        message: false
-        messageSubscription: false
+        message: true
+        messageSubscription: true
         variable: true
         variableDocument: true
         processInstance: true
         processInstanceCreation: false
```

Comment:
:x: not mentioned in the text above. Either add it (and others) and rephrase the text above that this is just an excerpt from the possible configuration options